README.rst

```diff
@@ -83,17 +83,26 @@ Examples
    # You can weight over the dimensions the function is being applied
    # to by passing the argument ``weights=weight`` with a xr.DataArray
    # containing the dimension(s) being reduced.
-   cos = np.abs(np.cos(np.arange(4)))
-   wgts = np.tile(cos, (5, 1)).reshape(4, 5)
-   wgt = xr.DataArray(
-      wgts,
-      coords=[
-         np.arange(4),
-         np.arange(5),
-      ],
-      dims=["lat", "lon"],
-   )
-   rmse_wgt = xs.rmse(obs, fct, ['lat', 'lon'], weights=wgt) 
+   base_data = np.ones((30,180,360))
+   a = xr.DataArray(base_data + np.random.rand(30,180,360), dims=['time','lat', 'lon'])
+   b = xr.DataArray(base_data + np.random.rand(30,180,360), dims=['time','lat', 'lon'])
+   x = np.linspace(-179.5, 179.5, 360)
+   y = np.linspace(-89.5, 89.5, 180)
+   lon, lat = np.meshgrid(x, y)
+   a['latitude'] = (('lat','lon'), lat)
+   a['longitude'] = (('lat','lon'), lon)
+   b['latitude'] = (('lat','lon'), lat)
+   b['longitude'] = (('lat','lon'), lon)
+
+   # make weights
+   weights = np.cos(np.deg2rad(a.latitude))
+   a, weights = xr.broadcast(a, weights)
+   DIM = ['lat', 'lon']
```

Comment:
Maybe move this up and replace ('lat', 'lon')

Reply:
Done!